### PR TITLE
pio-lib: add page

### PR DESCRIPTION
**Pull request by marchersimon**
For #5517

### CODE REVIEW

File: pages/common/pio-lib.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`pio lib list`
+
+- List built-in libraries based on installed development platforms and their frameworks:
+
+`pio lib buintin`
+
+- Search for existing libraries:
+
+`pio lib search {{library}}`
```
Comment (navarroaxel) [inline pages/common/pio-lib.md:16]: Is this search by exact name? or keywords?

Comment (marchersimon) [reply pages/common/pio-lib.md:16]: It's using keywords. How about if I change it to `{{keyword}}`?

File: pages/common/pio-lib.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Update installed libraries:
+
+`pio lib update`
+
+- Uninstall a library:
+
+`pio lib uninstall {{library}}`
+
+- Show PlatformIO library registry statistics:
```
Comment (sbrl) [inline pages/common/pio-lib.md:34]: Is this local stats or remote stats?

Comment (marchersimon) [reply pages/common/pio-lib.md:34]: As far as I know, there is only a remote registry.

File: pages/common/pio-lib.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/pio-lib.md]: @marchersimon go for it!

Comment (marchersimon) [reply pages/common/pio-lib.md]: Done (https://github.com/tldr-pages/tldr/pull/5728/commits/d8174eaa040d856ee04ff4bdde747e0e6f307a3a)

**Review by navarroaxel — APPROVED** [review]

File: pages/common/pio-lib.md (hunk 2 above)
Comment (Waples) [reply pages/common/pio-lib.md:34]: According to the [documentation](https://docs.platformio.org/en/v5.1.1/core/userguide/lib/cmd_stats.html), it shows the versions of the PlatformIO registry libs, i.e. remote.

File: pages/common/pio-lib.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+> Manage PlatformIO libraries.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/lib/>.
+
+- List installed libraries:
+
+`pio lib list`
+
+- List built-in libraries based on installed development platforms and their frameworks:
+
+`pio lib buintin`
```
Comment (bl-ue) [inline pages/common/pio-lib.md:12]:
```suggestion
`pio lib builtin`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**